pkg/controllers/dynakube/activegate/internal/authtoken/reconciler.go

```diff
@@ -29,18 +29,16 @@ const (
 var _ controllers.Reconciler = &Reconciler{}
 
 type Reconciler struct {
-	client    client.Client
-	apiReader client.Reader
-	dk        *dynakube.DynaKube
-	dtc       dtclient.Client
+	dk          *dynakube.DynaKube
+	dtc         dtclient.Client
+	secretQuery k8ssecret.QueryObject
 }
 
 func NewReconciler(clt client.Client, apiReader client.Reader, dk *dynakube.DynaKube, dtc dtclient.Client) *Reconciler {
 	return &Reconciler{
-		client:    clt,
-		apiReader: apiReader,
-		dk:        dk,
-		dtc:       dtc,
+		dk:          dk,
+		dtc:         dtc,
+		secretQuery: k8ssecret.Query(clt, apiReader, log),
```

Comment:
IMO it's better to call it `secret` or `secretObject` and since secret Object is a bit too much I prefer call it secret.

why? bc when you read carefully usage code (secret query get - sounds a bit odd):
```
r.secretQuery.Get(
```

but instead:

```
r.secret.Get(

r.secret.WithOwner(r.dk).Create
```
because "query" - says we're querying something, and on top getting or creating

Reply:
(another idea from @0sewa0 call it `secrets` not `secret`).